### PR TITLE
Remove parameter storage minimum level from cylindrical thermal storage

### LINKED ISSUE #1123: Remove minimum permissible storage volume from cylindrical thermal storage
To be in line with energy storage (see #942) the minimum permissible storage volume parameter should be removed. Thus this simplifies the calculation of the (usable) stored energy.

### PR DESCRIPTION
**Pull request by danielfeismann**
resolves #1123

### CODE REVIEW

File: src/test/groovy/edu/ie3/datamodel/io/factory/input/CylindricalStorageInputFactoryTest.groovy
Comment (sebastian-peter) [inline src/test/groovy/edu/ie3/datamodel/io/factory/input/CylindricalStorageInputFactoryTest.groovy:1]: Could also be removed from line 35

**Review by sebastian-peter — CHANGES_REQUESTED** [review]
Looks good, only have one tiny remark

File: src/test/groovy/edu/ie3/datamodel/io/factory/input/CylindricalStorageInputFactoryTest.groovy
Comment (danielfeismann) [reply src/test/groovy/edu/ie3/datamodel/io/factory/input/CylindricalStorageInputFactoryTest.groovy:1]: Good catch :)

**Review by sebastian-peter — APPROVED** [review]
LGTM

**Pull request merged**